pacstall

```diff
@@ -428,14 +434,7 @@ function is_apt_package_installed() {
 }
 
 function is_array() {
-    local input raw
-    input="${1}"
-    raw="$(declare -p "${input}" 2> /dev/null)"
-    if [[ ${raw} == "declare -a ${input}"* ]]; then
-        return 0
-    else
-        return 1
-    fi
+    [[ ${!1@a} == *a* ]]
```

Comment:
we should use this in https://github.com/pacstall/pacstall/blob/931763db64584a2798eaaecdd6d86aa1f2d7455b/misc/scripts/srcinfo.sh#L188 +
https://github.com/Elsie19/srcinfo_bash/blob/649141624f34844c280711de4a41c6a4be0f0faf/src/srcinfo.sh#L164 too